src/main/utils/database/EddDatabase.ts

```diff
@@ -508,10 +508,14 @@ class EddDatabase {
       ...downloadsData
     ]
 
-    if (!data.length) return null
+    if (!data.length) return [null, 0, []]
 
-    return this.db('pauses')
+    const insertedCount = await this.db('pauses')
       .insert(data)
+
+    const pausedDownloadIds = data.map((item) => item.downloadId)
+
+    return [insertedCount, insertedCount.length, pausedDownloadIds]
```

Comment:
Maybe return an object here 
```suggestion
    return {"insertedCount": insertedCount, ...}
    etc so its a bit cleaner than pulling indexes out when you parse the response
```